wagtail/admin/templates/wagtailadmin/shared/headers/page_edit_header.html

```diff
@@ -28,28 +27,10 @@ <h1 class="w-sr-only">
             </a>
         {% endif %}
 
-        {# Comments Toggle #}
-        {# TODO: Hookup comments number and self.show_comments_toggle #}
-        {% with comments_number=self.form.formsets.comments|length %}
-            <button
-                type="button"
-                class="{{ nav_icon_button_classes }} {% if comments_number %} w-mr-2.5 {% endif %}"
-                aria-label="{% trans 'Toggle comments' %}"
-                data-tippy-content="{% trans 'Comments' %}"
-                data-tippy-offset="[0, 0]"
-                data-tippy-placement="bottom"
-                data-comments-toggle
-            >
-                {% icon name="comment" class_name=nav_icon_classes %}
-
-                {% if comments_number %}
-                    <div class="w-w-[10px] w-h-[10px] w-flex w-justify-center w-items-center w-p-[3px] w-font-bold w-absolute w-bg-teal-100 w-text-white w-text-[10px] w-border w-border-white w-rounded-full -w-top-[10px] -w-right-[12px]"
-                        data-comments-toggle-count>
-                        {# populated in page-editor.js #}
-                    </div>
-                {% endif %}
-            </button>
-        {% endwith %}
+        {# Side panel Toggles #}
+        {% for panel in side_panels %}
+            {% component panel.toggle %}
```

Comment:
Could we make those toggles reuse the `nav_icon_classes` and `nav_icon_button_classes` defined on line 16? We need to make sure the buttons have consistent heights:

![button-heights](https://user-images.githubusercontent.com/877585/162922917-3aebfbaa-5d15-4c27-a85f-dd661924f40c.png)